r4/fhirpath/README.md

```diff
@@ -0,0 +1,99 @@
+## Code to show the usage of Fhirpath API
+
+```import ballerina/io;
+import wso2healthcare/healthcare.fhir.r4;
+import ballerina/http;
+
+public function main() returns error? {
+    http:Client fhirPathClient = check new ("Fhirpath_api_running_URL");
+
+    string [] fhirPath = ["Patient.address[1].line[0]","Patient.actve","Patient.address[0].line[0]","Patient.address.use"];
+
+    json payload = {
+        "fhirPath": fhirPath,
+        "fhirResource" : patient.toJson() };
+
+    http:Request request = new;
+    request.setJsonPayload(payload);
+    http:Response response = check fhirPathClient->/fhir/r4/fhirpath.post(request);
+
+    map<json>|error resultmap =response.getJsonPayload().ensureType();
+
+    if (resultmap is map<json>) {
+        
+        foreach string path in fhirPath{
+            json|error data = resultmap[path].result;
+
+            if data is json {
+                io:println("FHIRPath: ", path);
+                io:println("Result: ", data);
+                io:println();
+
+            } else {
+                io:println("FHIRPath: ", path);
+                io:println("Result Error: ", resultmap[path].resultenError);
+                io:println();
+            }
+        } 
+    } else {
+        io:println("Error");
+    }  
+}
+
+r4:Patient patient = {
+        "resourceType" : "Patient",
+        "id": "1",
+        "meta": {
+            "profile": [
+                "http://hl7.org/fhir/StructureDefinition/Patient"
+            ]
+        },
+        "active":true,
+        "name":[
+            {
+                "use":"official",
+                "family":"Chalmers",
+                "given":[
+                    "Peter",
+                    "James"
+                ]
+            },
+            {
+                "use":"usual",
+                "given":[
+                    "Jim"
+                ]
+            }
+        ],
+        "gender":"male",
+        "birthDate":"1974-12-25",
+        "managingOrganization":{
+            "reference":"Organization/1",
+            "display":"Burgers University Medical Center"
+        },
+        "address":[
+            {
+                "use":"home",
+                "line":[
+                    "534 Erewhon St",
+                    "sqw"
+                ],
+                "city":"PleasantVille",
+                "district":"Rainbow",
+                "state":"Vic",
+                "postalCode":"3999",
+                "country":"Australia"
+            },
+            {
+                "use":"work",
+                "line":[
+                    "33[0] 6th St"
+                ],
+                "city":"Melbourne",
+                "district":"Rainbow",
+                "state":"Vic",
+                "postalCode":"3000",
+                "country":"Australia"
+            }
+        ]
+    };```
```

Comment:
Let's add steps on how to run the program and sample output as well in the README

Reply:
Fixed in 28ab50418e064931acc8a5e17b22cbfa773c827a